### DISCUSSION TITLE
Is there a way to add comments to the Localization Strings?

### DISCUSSION THREAD

**Discussion by mxnmike** [SwiftGen/SwiftGen#862] [Q&A] [answered]
Hi, Im trying to add comments to my localization strings so when I need to support a new language, translators will get an idea of for what that text is used for?

Thanks in Advance.

**Comment by djbe** [comment, marked as answer]
Check the Apple docs about strings files for how they work:
https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/LoadingResources/Strings/Strings.html

Essentially:
```json
/* Insert Element menu item */
"menu.item.insert" = "Insert Element";

/* Error string used for unknown error types. */
"error.unknown" = "An unknown error occurred.";
```
